docs/openfaas-pro/sns-events.md

```diff
@@ -0,0 +1,89 @@
+# Trigger functions from AWS SNS messages
+
+With the SNS connector AWS SNS subscriptions can be used to trigger OpenFaaS functions.
+
+> Note: This feature is included for [OpenFaaS Pro](https://openfaas.com/support/) customers.
+
+## Installation
+
+### Pre-reqs on AWS
+
+* Create a SNS topic
+    The connector can only be used with topics Standard topics and not FIFO topics. Make sure your topic uses the right type.
+
+* Save the topic ARN
+
+
+### Install the connector with Helm
+
+* Set up AWS
+    You can configure permissions using a dedicated IAM user, or if your cluster is configured for AWS IAM, ambient credentials mapped into the Pod at runtime.
+
+* Set up the connector
+    The SNS connector can be installed using its [helm chart](https://github.com/openfaas/faas-netes/tree/master/chart/sns-connector)
+
+
+* Configure the connector for you needs
+        
+    `callbackURL` - Public callback URL for subscriptions
+
+    `topicARN` - SNS topic ARN
+
+    `awsRegion` - The AWS region for your SNS topics
+
+> Note that the callback URL has to be publicly accessible to receive SNS messages. An ingress configuration example is available in the [chart README](https://github.com/openfaas/faas-netes/tree/master/chart/sns-connector#configure-ingress)
+
+## Usage
+
+Once you configured the connector to receive callbacks for an SNS topic you can annotate you functions so they get triggered by any incoming message on that topic.
+
+```bash
+faas-cli deploy \
+    --name printer \
+    --annotation topic=arn:aws:sns:us-east-1:222943518635:of-event
+```
+
+The printer function will print out the body and headers of each request which makes it a great function for debugging events.
+
+You can publish a message to your topic from the AWS console. 
+
+Example output of the printer function:
+
+```bash
+faas-cli logs printer
+
```

Comment:
Take out the Forwarded host, forwarded for and inlets messages which don't make sense here.